sdk/tests/conformance/extensions/webgl-multi-draw.html

```diff
@@ -846,10 +847,181 @@
   }
 }
 
-runTest();
+function waitForComposite() {
+  debug('wait for composite');
+  return new Promise(resolve => wtu.waitForComposite(resolve));
```

Comment:
I have not seen it be intermittent but ... the canvas has to be visible. That's why I set it to `position: fixed; top: 0`, so that regardless of where the page is scrolled the canvas is visible.  Before I added that it would not repo if offscreen.